pyaedt/modeler/Object3d.py

```diff
@@ -1678,6 +1681,56 @@ def unite(self, object_list):
         self._primitives.modeler.unite(unite_list)
         return self
 
+    @aedt_exception_handler
+    def rotate(self, cs_axis, angle=90.0, unit="deg"):
+        """Rotate the selection.
+
+        Parameters
+        ----------
+        cs_axis
+            Coordinate system axis or the Application.CoordinateSystemAxis object.
+        angle : float, optional
+            Angle of rotation. The units, defined by ``unit``, can be either
+            degrees or radians. The default is ``90.0``.
+        unit : text, optional
+             Units for the angle. Options are ``"deg"`` or ``"rad"``.
+             The default is ``"deg"``.
+
+        Returns
+        -------
+        bool
+            ``True`` when successful, ``False`` when failed.
+
+        References
+        ----------
+
+        >>> oEditor.Rotate
+        """
+        return self._primitives.modeler.rotate(self.id, cs_axis=cs_axis, angle=angle, unit=unit)
+
+    @aedt_exception_handler
+    def move(self, vector):
+        """Move objects from a list.
+
+        Parameters
+        ----------
+        objid : list, Position object
+            List of object IDs.
```

Comment:
@maxcapodi78 I don't think it is used anymore. It seems that you use `self.id` now. But just to be sure.
```suggestion
```